ReviewBot.py

```diff
@@ -408,7 +408,11 @@ def devel_project_review_needed(self, request, project, package):
         if len(maintainers) == 0:
             devel_project, devel_package = devel_project_fallback(self.apiurl, project, package)
             if devel_package:
-                maintainers = set(package_role_expand(self.apiurl, devel_project, devel_package))
+                try:
+                    maintainers = set(package_role_expand(self.apiurl, devel_project, devel_package))
+                except HTTPError:
+                    self.logger.debug('Cannot find maintainer information')
+                    return False
```

Comment:
Is it a good idea to have `False` as a fallback here? Spurious failures would result in a missing review.

Reply:
well, not returning false wants to add a maintainer which we cannot identify, as we have no devel project.

What could make sense is ensure we get a 404 - which is unlikely to happen when we get a server error (500 / 403)